### PR TITLE
stolostron/backlog#24683 rem ZTP tp

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @berenss . Looks like this went GA. Does this look ok? Thanks!

File: clusters/single_node_openshift.adoc [hunk 1]
```diff
@@ -3,9 +3,7 @@
...
 
-This procedure explains how to deploy a single SNO cluster. You can also test the procedure of provisioning multiple SNO clusters on edge resources by using the zero touch provisioning, which is a Technology Preview feature. For more information about that procedure, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation. 
-
-*Important:* Creating a SNO cluster should not be used in production environments.
+This procedure explains how to deploy a single SNO cluster. You can also test the procedure of provisioning multiple SNO clusters on edge resources by using the zero touch provisioning feature. For more information about that procedure, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation.
```
Comment (berenss) [inline clusters/single_node_openshift.adoc]: Let's go with:
You can also use the zero touch provisioning procedure which enables the provisioning of multiple SNO clusters, suitable for large scale edge deployments

**Comment by berenss** [comment]
thank you Chris! any guess (doesn't need to be exact) when it will go live in the 2.5 public doc portal?  and no need to respond today (Friday)

File: clusters/single_node_openshift.adoc [hunk 2]
```diff
@@ -3,9 +3,7 @@
...
 
-This procedure explains how to deploy a single SNO cluster. You can also test the procedure of provisioning multiple SNO clusters on edge resources by using the zero touch provisioning, which is a Technology Preview feature. For more information about that procedure, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation. 
-
-*Important:* Creating a SNO cluster should not be used in production environments.
+This procedure explains how to deploy a single SNO cluster. You can also use the zero touch provisioning procedure to provision multiple SNO clusters, which is suitable for large scale edge deployments. For more information about that procedure, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation.
```
Comment (dockerymick) [inline clusters/single_node_openshift.adoc]: A single, Single Node OpenShift cluster?

This procedure explains how to deploy a single SNO cluster. You can also use the zero-touch provisioning procedure to provision multiple SNO clusters, which is suitable for large-scale edge deployments. For more information about that procedure, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation.

**Review by dockerymick — APPROVED** [review]
Made a comment, thanks for pulling me in

File: clusters/single_node_openshift.adoc (hunk 2 above)
Comment (berenss) [reply clusters/single_node_openshift.adoc]: This procedure explains how to deploy a SNO cluster

File: clusters/single_node_openshift.adoc [hunk 3]
```diff
@@ -3,9 +3,7 @@
 
 You can create a single node OpenShift (SNO) cluster that contains only a single node. In a standard cluster, there are control plane nodes and separate worker nodes. This configuration can be helpful when you have many clusters that need to minimize their resource footprint.
```
Comment (berenss) [inline clusters/single_node_openshift.adoc:5]: In a typical cluster, there are control plane nodes and separate worker nodes. To optimize your deployment model and minimize the resource footprint, it can be useful to create a single node OpenShift (SNO) cluster that contains only a single node which comprises both the control plane and worker as one node.
